tests/test_unique_queues.py

```diff
@@ -368,40 +368,57 @@ async def test_drop_all_items():
     await deactivate_test(client)
 
 
+@pytest.fixture
+def patch_time(monkeypatch):
```

Comment:
If you move the fixture to the file `/tests/conftest.py` you can use it everywhere in tests without code duplication ;-) 
https://towardsdatascience.com/testing-best-practices-for-machine-learning-libraries-41b7d0362c95